new-backoffice/tutorials/creating-a-custom-dashboard/adding-functionality-to-the-dashboard.md

```diff
@@ -108,28 +104,24 @@ Your dashboard should now look something like this:
 
 {% code title="welcome-dashboard.element.ts" lineNumbers="true" %}
 ```typescript
-import { UMB_AUTH, UmbLoggedInUser } from '@umbraco-cms/backoffice/auth';
+import { type UmbCurrentUserModel, UMB_CURRENT_USER_CONTEXT } from "@umbraco-cms/backoffice/current-user";
 import { LitElement, css, html, customElement, state } from "@umbraco-cms/backoffice/external/lit";
 import { UmbElementMixin } from "@umbraco-cms/backoffice/element-api";
 
 @customElement('my-welcome-dashboard')
```

Comment:
I don't know what the difference is you are talking about here. This line wasn't changed in my PR. Are you strictly referring to the quotes? If that is the case, we should just figure out what we want to do since either works in javascript.

Reply:
yes, it is the quotes I was referring to 🙈 I can see in some examples I had with double quotes while the default ones use single quotes, so I will make sure it is on all examples with single quotes 💪